lib/routemap_cli.c

```diff
@@ -948,7 +948,7 @@ DEFPY(
 
 DEFPY(
 	no_rmap_call, no_rmap_call_cmd,
-	"no call",
+	"no call [NAME]",
```

Comment:
We are not going to use name in NB call ? We are just having it in CLI but NB is being passed with NULL.

Reply:
there is no need.  There is only 1 call function per section and we are removing it right?

Reply:
That is right. Right now only 1 call function per section is allowed and it overwrites if we try to add more. I was thinking by bringing this in you are trying to add more than one call per routemap sequence in future if that is not the case and it is only command symmetry then this looks good.